ddl/migrations/redshift/1458766468.sql

```diff
@@ -0,0 +1,31 @@
+BEGIN;
+
+ALTER TABLE activity_stream_stats_daily ADD COLUMN date DATE;
+UPDATE activity_stream_stats_daily SET date = TRUNC(receive_at);
```

Comment:
@tspurway Yup, it's already been populated here. Redshift can't take Alembic migrations.